content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -54,6 +59,10 @@ The Windows Server Host Operating System is subject to the [Windows Server ](htt
 Windows containers with process isolation have strict compatibility rules, [where the host OS version must match the container base image OS version](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility). Once we support Windows containers with Hyper-V isolation in Kubernetes, the limitation and compatibility rules will change.
 {{< /note >}}
 
+#### Compute
+
+From an API and kubectl perspective, Windows containers behave in much the same way as Linux-based containers. However, there are some notable differences in key functionality which are outlined in the limitation section.
```

Comment:
nit: hyperlink to the list of limitations.

Reply:
thanks! resolved